modules/microshift-restart-ovnkube-master.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-networking.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-restart-ovnkube-master_{context}"]
+= Restarting the ovnkube-master pod
+
+The following procedure restarts the ovn-kube master pod.
```

Comment:
In the case this file is needed, let's use the name `ovnkube-master pod` everywhere.